src/main/java/donnafin/storage/JsonAdaptedPerson.java

```diff
@@ -36,11 +38,17 @@
     @JsonCreator
     public JsonAdaptedPerson(@JsonProperty("name") String name, @JsonProperty("phone") String phone,
             @JsonProperty("email") String email, @JsonProperty("address") String address,
-            @JsonProperty("tagged") List<JsonAdaptedTag> tagged) {
+            @JsonProperty("notes") String notes, @JsonProperty("tagged") List<JsonAdaptedTag> tagged) {
         this.name = name;
         this.phone = phone;
         this.email = email;
         this.address = address;
+        if (notes == null) {
+            this.notes = " ";
+        } else {
+            this.notes = notes;
+
+        }
```

Comment:
Do we need an empty space?

```suggestion
        if (notes == null) {
            this.notes = "";
        } else {
            this.notes = notes;
        }
```